app/validators/poll_validator.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class PollValidator < ActiveModel::Validator
+  MAX_OPTIONS      = 4
+  MAX_OPTION_CHARS = 25
```

Comment:
Those two limits seem quite arbitrary. `4` for `MAX_OPTIONS` sounds a bit low to be honest.
Overall this should be fine for local statuses, but when interacting with others instances, this could be an issue.
We should at the very check what limits other implementations have, if any.

Reply:
4 is the max on Twitter, and I am inclined to agree. Anything else turns your home feed into a survey.

@syuilo I can't find validations in Misskey, what limits are there?

Reply:
In Misskey, we can create up to 10 choices which have 50 characters or less. However, there is no limitations like that in notes which created by ActivityPub (maybe. I'll test about that).

Reply:
Okay, I think I will have soft limits for local polls (4) and higher hard limits (10) for remote ones.